packages/bpk-component-checkbox/src/__snapshots__/BpkCheckbox-test.js.snap

```diff
@@ -78,6 +78,7 @@ exports[`BpkCheckbox should render correctly with checked attribute 1`] = `
     <input
       aria-invalid="false"
       aria-label="Prefer directs"
+      checked=""
```

Comment:
Yes, I noticed it at the first time, and had reverted, but I found that when I tested it, it would report an error, it told me that these four files must be updated. I think the only explanation is most likely that I modified the React version.

Reply:
I checkout another PR for testing, I found that after upgrading to React v16.8+, in a structure similar to 
`<label><input/></label>`, the input will be automatically added with the checked attribute. 
This seems to make sense, since the components that contain these structures are indeed all related to checked🤔.